zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/dto/SearchResponseDto.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller.usermanagement.dto;
+
+import java.util.List;
+
+public class SearchResponseDto<T> {
```

Comment:
I like the implementation of search response. I assume that this type of class should actually already exist on the global URA level, do you know if this is the case?

Reply:
Yes, I have asked about that in general ... in fact the usual approach so far for gateway-rest was to define the endpoints in an openapi yaml file and during the mvn clean install -Dquickly this will generate the DTO classes defined in this yaml ... 

Unfortunately, I haven't managed to make this generate a class using a Generic as above, which is what I will look at asap now. I will ask for help with that again...